parts/base/src/test/java/xxxxxx/yyyyyy/zzzzzz/selenium/FunctionTestSupport.java

```diff
@@ -0,0 +1,186 @@
+/*
```

Comment:
- [ ] Are there any duplicates?
Since we do not expect to distribute test materials, only the testScript distribution is sufficient.

Reply:
https://github.com/terasolunaorg/terasoluna-gfw-web-blank/pull/579/commits/dcaadcbe98e6a087dac6f0518ca5a84978184010
Unnecessary material has been removed with this commit.